examples/stream/stream.cpp

```diff
@@ -211,16 +263,29 @@ int main(int argc, char ** argv) {
             return 1;
         }
     }
-
-    printf("[Start speaking]");
+    // save wav file
+    SimpleWavWriter *wavWriter = nullptr;
+    if (params.save_audio) {
+        // Get current date/time for filename
+        time_t now = time(0);
+        char buffer[80];
+        strftime(buffer, sizeof(buffer), "%Y%m%d%H%M%S", localtime(&now));
+        std::string filename = std::string(buffer) + ".wav";
+
+        wavWriter = new SimpleWavWriter(filename, WHISPER_SAMPLE_RATE, 16, 1);
+    }
+    printf("[Start speaking]\n");
     fflush(stdout);
 
-          auto t_last  = std::chrono::high_resolution_clock::now();
+    auto t_last  = std::chrono::high_resolution_clock::now();
     const auto t_start = t_last;
 
     // main audio loop
     while (is_running) {
         // handle Ctrl + C
```

Comment:
```suggestion
```

Reply:
What to do with this place? Do you have any suggestions?

Reply:
> What to do with this place? Do you have any suggestions?

I first removed this comment, and then moved it down to the right place.